service.subtitles.napisy24pl/service.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+from urllib.parse import unquote
+
+try:
+    import xbmc
+    import xbmcvfs
+    import xbmcaddon
+    import xbmcplugin
+    import xbmcgui
+except ImportError:
+    from tests.stubs import xbmc, xbmcgui, xbmcaddon, xbmcplugin, xbmcvfs
+
+__addon__ = xbmcaddon.Addon()
+__scriptid__ = __addon__.getAddonInfo('id')
+__scriptname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__language__ = __addon__.getLocalizedString
+
+__cwd__ = xbmc.translatePath(__addon__.getAddonInfo('path'))
+__profile__ = xbmc.translatePath(__addon__.getAddonInfo('profile'))
+__resource__ = xbmc.translatePath(os.path.join(__cwd__, 'resources', 'lib'))
+__temp__ = xbmc.translatePath(os.path.join(__profile__, 'temp', ''))
+
+sys.path.append(__resource__)
```

Comment:
please do not use `sys.path.append` and use absolute imports.

`from resources.lib import x,y,z`